prime-router/src/main/kotlin/fhirengine/utils/CompareFhirData.kt

```diff
@@ -276,9 +276,9 @@ class CompareFhirData(
         }
         if (!primitiveResult.passed) {
             val msg = if (expectedPrimitive is PrimitiveType<*>) {
-                val actualPrimitivesValues = actualPrimitive
-                    .filterIsInstance<PrimitiveType<*>>()
+                val actualPrimitivesValues = actualPrimitive.filterIsInstance<PrimitiveType<*>>()
                     .joinToString { it.asStringValue() }
+
```

Comment:
Nit: if a file has nothing but whitespace or simple comment changes like this, don't update it so we can keep the PR as specific to the ticket as possible.